<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoViewer</class>
 <widget class="QDialog" name="InfoViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="crystalTab">
      <attribute name="title">
       <string>Crystal</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CrystalInfoDocument" name="crystalInfoDocument" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="atomsTab">
      <attribute name="title">
       <string>Atoms</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="AtomInfoDocument" name="atomInfoDocument" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="surfaceTab">
      <attribute name="title">
       <string>Surface</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="SurfaceInfoDocument" name="surfaceInfoDocument" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="energiesTab">
      <attribute name="title">
       <string>Energies</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="InteractionInfoDocument" name="interactionsInfoDocument" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AtomInfoDocument</class>
   <extends>QWidget</extends>
   <header>atominfodocument.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CrystalInfoDocument</class>
   <extends>QWidget</extends>
   <header>crystalinfodocument.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InteractionInfoDocument</class>
   <extends>QWidget</extends>
   <header>interactioninfodocument.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SurfaceInfoDocument</class>
   <extends>QWidget</extends>
   <header>surfaceinfodocument.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
